src/lambda_codebase/initial_commit/bootstrap_repository/example-adfconfig.yml

```diff
@@ -8,13 +8,13 @@ regions:
 
 config:
   main-notification-endpoint:
-    - type: email # slack or email
-      target: jon.doe@email.com # Email/Slack channel who receives notifications for the main bootstrapping pipeline
+    - type: email  # Options include: slack or email
+      target: john.doe@example.com  # Email/Slack channel who receives notifications for the main bootstrapping pipeline
   moves:
     - name: to-root
       action: safe  # Can be safe or remove-base (see docs)
   scp:
-    keep-default-scp: enabled  # determines if the default AWSFullAccess SCP stays attached to all OU's
-  scm: # Source control management
-    auto-create-repositories: enabled # if true and using codecommit as source the repository will be automatically created
-    default-scm-branch: master        # allow to specify custom default branch if a pipeline specific configuration is not provided. If the parameter is not specified it defaults to the "master" branch.
+    keep-default-scp: enabled  # Determines if the default AWSFullAccess SCP stays attached to all OU's
+  scm:  # Source control management
+  auto-create-repositories: enabled  # If true and using CodeCommit as source, the repository will be automatically created
+    default-scm-branch: master       # The default branch is used when the pipeline does not specify a specific branch. If this parameter is not specified, it defaults to the "master" branch.
```

Comment:
Should this be main instead of master now?

Reply:
It will be, but not as part of this PR :smile: